modules/manage/pages/disaster-recovery/shadowing/setup.adoc

```diff
@@ -137,79 +150,7 @@ This creates a complete YAML configuration file that you can customize for your
-  retry_backoff_ms: 100               # Backoff between retries (default: 100ms)
-  fetch_wait_max_ms: 500              # Max time to wait for fetch requests (default: 500ms)
-  fetch_min_bytes: 5242880            # Min bytes per fetch (default: 5MB)
-  fetch_max_bytes: 20971520           # Max bytes per fetch (default: 20MB)
-  fetch_partition_max_bytes: 1048576  # Max bytes per partition fetch (default: 1MB)
-
-topic_metadata_sync_options:
-  interval: 30s                       # How often to sync topic metadata (examples: "30s", "1m", "5m")
-  auto_create_shadow_topic_filters:   # Filters for automatic topic creation
-  - pattern_type: LITERAL             # Include all topics (wildcard)
-    filter_type: INCLUDE
-    name: '*'
-  - pattern_type: PREFIX              # Exclude topics with specific prefix
-    filter_type: EXCLUDE
-    name: <topic-prefix-to-exclude>   # Examples: "temp-", "test-", "debug-"
-  synced_shadow_topic_properties:     # Additional topic properties to sync (beyond defaults)
-  - retention.ms                      # Topic retention time
-  - segment.ms                        # Segment roll time
-  exclude_default: false              # Include default properties (compression, retention, etc.)
-  start_at_earliest: {}               # Start from the beginning of source topics (default)
-  paused: false                       # Enable topic metadata synchronization
-
-consumer_offset_sync_options:
-  interval: 30s                       # How often to sync consumer group offsets
-  paused: false                       # Enable consumer offset synchronization
-  group_filters:                      # Filters for consumer groups to sync
-  - pattern_type: LITERAL
-    filter_type: INCLUDE
-    name: '*'                         # Include all consumer groups
-
-security_sync_options:
-  interval: 30s                       # How often to sync security settings
-  paused: false                       # Enable security settings synchronization
-  acl_filters:                        # Filters for ACLs to sync
-  - resource_filter:
-      resource_type: TOPIC            # Resource type: "TOPIC", "GROUP", "CLUSTER"
-      pattern_type: PREFIXED          # Pattern type: "LITERAL", "PREFIXED"
-      name: <resource-pattern>        # Examples: "prod-", "app-data-"
-    access_filter:
-      principal: User:<username>      # Principal name, example: "User:app-service"
-      operation: ANY                  # Operation: "READ", "WRITE", "CREATE", "DELETE", "ALTER", "DESCRIBE", "ANY"
-      permission_type: ALLOW          # Permission: "ALLOW" or "DENY"
-      host: '*'                       # Host pattern, examples: "*", "10.0.0.0/8", "app-server.example.com"
-
-schema_registry_sync_options:         # Schema Registry synchronization options
-  shadow_schema_registry_topic: {}    # Enable byte-for-byte _schemas topic replication
+include::manage:partial$example-generate-shadow-config.adoc[]
```

Comment:
This sample config file is nested under rpk, but I think it should be visible for the API config too, right?